sample-runner/shared/EnactLiveEdit.module.less

```diff
@@ -1,4 +1,4 @@
-// EnactLiveEdit.less
+// EnactLiveEdit.module.less
```

Comment:
This change means that `.error` and `.sandbox` are not for global scope. Do I understand correctly?